.travis.yml

```diff
@@ -8,8 +8,8 @@ notifications:
   email: false
 before_script:
 - npm prune
-- lerna bootstrap --scope '@(function-tree|cerebral|cerebral-forms|cerebral-provider-http|cerebral-provider-firebase|cerebral-provider-forms|cerebral-router|babel-*)'
-- lerna bootstrap --scope @cerebral/codemods*
+- npm run setup:babel
+- lerna bootstrap --scope cerebral-codemods*
```

Comment:
So if i get it right: you have renamed private packages to simplify bootstrap globs?